app/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM python:3-slim
 # Keep container packages up-to-date.
 RUN apt update \
     && apt upgrade --yes
+RUN apt-get install --no-install-recommends --yes postgresql libpq-dev
```

Comment:
I don't think we need this line. The API seems to run just fine and connect to the database without it.

Reply:
Ah, added that in an attempt to solve the M1 mac issue as those libraries it was complaining about, but realized the docker-compose change was the right fix. Removed.